src/pages/StudentRegistration/index.tsx

```diff
@@ -0,0 +1,629 @@
+  const years = Array.from({ length: currentYear - 1980 + 1 }, (_, i) => currentYear - i);
+
+  const [personal, setPersonal] = useState({
+    nome: '',
+    nascimento: '',
+    sexo: '',
+    raca: '',
+    escolaridade: '',
+    escolaAtual: '',
+    endereco: '',
+    cep: '',
+    numero: '',
+    complemento: '',
+    ano: '',
+    documentos: '',
+    identidade: '',
+    numeroIdentidade: ''
+  });
+
+  type Documento = {
+    arquivo: string;
+    tipo: string; // extensão
+    tipoDocumento: string; // tipo selecionado pelo usuário
+    origem: string;
+    data: string;
+    descricao: string;
+    id: number;
+  };
+
+  const [documentos, setDocumentos] = useState<Documento[]>([]);
+  const [showUploader, setShowUploader] = useState(false);
+  const [docForm, setDocForm] = useState({
+    arquivo: '',
+    tipo: '',
+    origem: '',
+    data: '',
+    descricao: '',
+  });
+  const [detalhes, setDetalhes] = useState({
+    dataCadastro: '',
+    comoConheceu: '',
+    vinculo: '',
+  });
+  const navigate = useNavigate();
+
+  const handleAddDoc = () => {
+    if (!docForm.arquivo) return;
+  
+    setDocumentos((docs) => [
+      ...docs,
+      {
+        ...docForm,
+        id: Date.now(),
+        tipo: personal.documentos,
+      } as Documento,
+    ]);
+  
+    setDocForm({ arquivo: '', tipo: '', origem: '', data: '', descricao: '' });
+    setPersonal((p) => ({ ...p, documentos: '' }));
+    setShowUploader(false);
+  };
+
+  return (
+    <Box
+      width="100%"
+      p={{ xs: 2, md: 4 }}
+      display="flex"
+      justifyContent="center"
+    >
+      <Box>
+        <Grid
+          container
+          spacing={{ xs: 3, md: 4 }}
+          alignItems="flex-start"
+          justifyContent="space-between"
+          sx={{
+            flexWrap: { xs: 'wrap', md: 'nowrap' },
+            '& > .MuiGrid-item': {
+              flexBasis: { md: '33.5%' },
+              maxWidth: { md: '33.5%' },
+            },
+            '@media (min-width:1200px)': {
+              '& > .MuiGrid-item': {
+                flexBasis: '35%',
+                maxWidth: '35%',
+              },
+            },
+            '@media (min-width:1536px)': {
+              '& > .MuiGrid-item': {
+                flexBasis: '36%',
+                maxWidth: '36%',
+              },
+            },
+            columnGap: { md: 3 },
+          }}
+        >
+          {/* Informações Pessoais */}
+          <Grid>
+            <Typography
+              variant="h6"
+              fontWeight="bold"
+              sx={{
+                mb: 2,
+                mt: 1,
+                color: 'primary.main',
+                borderBottom: 2,
+                borderColor: 'primary.main',
+              }}
+            >
+              Informações Pessoais
+            </Typography>
+
+            <TextField
+              label="Nome"
+              fullWidth
+              margin="normal"
+              placeholder="Digite o nome do Educando"
```

Comment:
Should come from `constants` file - as all strings
```suggestion
              placeholder={pt.studentRegistration.form.namePlaceholder}
```